src/main/kotlin/org/openmbee/flexo/mms/routes/ldp/RepoRead.kt

```diff
@@ -18,29 +18,10 @@ private val SPARQL_BGP_REPO: (Boolean, Boolean) -> String = { allRepos, allData
                 mms:etag ?__mms_etag ;
                 mms:org ?$SPARQL_VAR_NAME_ORG ;
                 ${"?repo_p ?repo_o ;" iff allData}
-                .
-                
-            bind(iri(concat(str(?$SPARQL_VAR_NAME_REPO), "/graphs/Metadata")) as ?metadataGraph)
+                .                
         """.reindent(if(allRepos) 3 else 2)}
         ${"}" iff allRepos}
-
-        ${"""
```

Comment:
these shouldn't be removed imo

Reply:
It doesn't really matter here since nothing would match that anyways. I actually don't think it should be returning these triples even if they exist - part of #159